core/CMakeLists.txt

```diff
@@ -632,14 +651,7 @@ set(have_crypto "${HAVE_CRYPTO}")
 set(have_openssl "${HAVE_OPENSSL}")
 set(have_zlib "${ZLIB_FOUND}")
 
-set(HAVE_CAPABILITY
-    "${HAVE_CAPABILITY_H}"
-    CACHE BOOL "compile with capability support"
-)
-if("${HAVE_CAPABILITY}")
-  set(CAPABILITY_LIBS cap)
-  message(STATUS "CAPABILITY set")
-endif()
+option(ENABLE_CAPABILITY "Enable capability support" "${HAVE_CAPABILITY_H}")
```

Comment:
```suggestion
option(ENABLE_CAPABILITY "Enable capability-based security support" "${HAVE_CAPABILITY_H}")
```

Reply:
Right now this enables/disables support for the `-k` option on the FD.
Maybe we should call it "Enable keep readall capabilities in filedaemon", as that is what this actually switches.

Reply:
If it is only used in the FD: yes, that would be helpful.